Apps/Medication/src/Properties/launchSettings.json

```diff
@@ -1,23 +1,23 @@
 {
-    "iisSettings": {
-        "windowsAuthentication": false,
-        "anonymousAuthentication": true,
-        "iisExpress": {
-            "applicationUrl": "http://0.0.0.0:3003",
-            "sslPort": 0
-        }
-    },
-    "$schema": "http://json.schemastore.org/launchsettings.json",
-    "profiles": {
-        "HealthGateway.Medication": {
-            "commandName": "Project",
-            "launchUrl": "swagger",
-            "applicationUrl": "http://0.0.0.0:3003;",
-            "environmentVariables": {
-                "ODRPROXY_SERVICE_PORT": "443",
-                "ODRPROXY_SERVICE_HOST": "dev-odrproxy.pathfinder.gov.bc.ca",
-                "ASPNETCORE_ENVIRONMENT": "Development"
-            }
-        }
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://0.0.0.0:3003",
+      "sslPort": 0
     }
-}
+  },
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "HealthGateway.Medication": {
+      "commandName": "Project",
+      "launchUrl": "swagger",
+      "applicationUrl": "http://0.0.0.0:3003;",
+      "environmentVariables": {
+        "ODRPROXY_SERVICE_PORT": "443",
+        "ODRPROXY_SERVICE_HOST": "dev-odrproxy.pathfinder.gov.bc.ca",
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
+  }
+}
```

Comment:
Similar to the commas at the end of lists, prettier enforces our .editorconfig settings which specify an indentation size of 4 for all files except .yml and .yaml.  This can be auto-resolved with prettier.

Reply:
not sure why this happened here only, any thoughts? it seems like everywhere else is fine